src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -216,29 +216,6 @@ public void Deconstruct(out ParameterSymbol? parameter, out BoundExpression argu
                 ? p.ToString()
                 : Argument.ToString();
         }
-
-        /// <summary>
```

Comment:
It feels like this comment could be preserved on `Lifetime`

Reply:
I tried to migrate pieces of this comment to the appropriate members of Lifetime. For example, the lines which are describing the special scopes, are now moved to the properties for accessing those special scopes.